index.html

```diff
@@ -258,6 +321,28 @@
       _p.cmd.viewport(width-gui_width,height)
       _p.draw()`
 					      );
+            cv.addEventListener('click', onClick);
+            document.addEventListener('mousemove', onMouseMove);
+            document.addEventListener('mousemove', onMouseMove4Stop);
+            //cv.addEventListener('mousestop', onMouseStop);
+            cv.addEventListener('mousedown', onMouseDown);
+            cv.addEventListener('mouseup', onMouseUp);
+            cv.addEventListener('contextmenu', onContextMenu);
+            //cv.addEventListener('dblclick', onDoubleClick);
+            document.addEventListener('keydown', onKeyDown);
+            document.addEventListener('keyup', onKeyUp);
+            document.addEventListener("wheel", (event) => {
+                if (event.wheelDeltaY < 0) {
+                    onWheelRollUp(event);
+                } else if (event.wheelDeltaY > 0) {
+                    onWheelRollDown(event);
+                }
+            })
+        } else {
```

Comment:
Skipped pymol setup process.
This will allow loading additional files after loading first PDB file.